go.mod

```diff
@@ -2,19 +2,19 @@ module github.com/crc-org/machine-driver-libvirt
 
 require (
 	github.com/crc-org/machine v0.0.0-20221028075518-f9b43442196b
-	github.com/libvirt/libvirt-go v4.5.0+incompatible
-	github.com/sirupsen/logrus v1.9.0
+	github.com/libvirt/libvirt-go v7.4.0+incompatible
```

Comment:
ohh, is it for building the driver, or on hosts that want to run crc we need to check this as part of preflight?

Reply:
if we build this with a too new libvirt-go, then `crc` won't start on machines with older libirt, so this cannot easily be checked with a preflight check.
It might be improved with newer libvirt-go versions though, see https://github.com/openshift/installer/pull/7252#discussion_r1242097763 and https://github.com/openshift/installer/pull/7305

Reply:
rhel8 has libvirt-8.0.0 packages now

Reply:
Looks like rhel8.6 and newer have libvirt 8.0

Reply:
in rhel8 libvirt 8.0.0 is available now, but to keep support for distros with older libvirt we should keep using the older version for now, in that case i'll update the other libraries and the go version to `1.19` and keep `libvirt-go` at `v4.5.0`

Reply:
> but to keep support for distros with older libvirt we should keep using the older version for now,

Which distros are we talking about?

Reply:
i was thinking of debian, but debian 12 has newer libvirt 9.0, debian 11 has old 7.0

Reply:
debian stable has 9.0.0, oldstable has 7.0.0
ubuntu 22.04 LTS has 8.0.0 and 20.04 LTS has 6.0.0.

So far we've tried to support the last 2 LTS releases of distros, or the last 2 releases of faster moving distros. Imo we could raise our requirements to at least 6.0

Reply:
jammy-updates (20.04 LTS) has a 8.0.0 package

Reply:
updated libvirt-go to 6.0.0